SensusUI/Inputs/NumberSliderInput.cs

```diff
@@ -172,48 +240,69 @@ public override View GetView(int index)
                     {
                         _incrementalValue = newIncrementalValue;
                         _incrementalValueHasChanged = true;
-                        _sliderLabel.Text = GetLabelText(index) + "  " + _incrementalValue;
+                        _sliderLabel.Text = DisplaySliderValue ? GetLabelText(index) + "  " + _incrementalValue : GetLabelText(index) + "  " + _tipText;
                         Complete = Value != null;
                     }
                 };
 
-                base.SetView(new StackLayout
-                    {
-                        Orientation = StackOrientation.Vertical,
-                        VerticalOptions = LayoutOptions.Start,
-                        Children =
-                        { 
-                            _sliderLabel,
-                            new StackLayout
+                StackLayout stack = new StackLayout();
+                stack.Orientation = StackOrientation.Vertical;
+                stack.VerticalOptions = LayoutOptions.Start;
+                stack.Children.Add(_sliderLabel);
```

Comment:
Does this look ok?

                base.SetView(new StackLayout
                    {
                        Orientation = StackOrientation.Vertical,
                        VerticalOptions = LayoutOptions.Start,
                        Children =
                            {
                                _sliderLabel,
                                new StackLayout
                                {
                                    Orientation = StackOrientation.Horizontal,
                                    HorizontalOptions = LayoutOptions.FillAndExpand,
                                    Children =
                                        {
                                            new Label
                                            {
                                                Text = _minimum.ToString(),
                                                FontSize = 20,
                                                HorizontalOptions = LayoutOptions.Fill,
                                                IsVisible = _displayMinMax ? true : false
                                            },
                                            _slider,
                                            new Label
                                            {
                                                Text = _maximum.ToString(),
                                                FontSize = 20,
                                                HorizontalOptions = LayoutOptions.Fill,
                                                IsVisible = _displayMinMax ? true : false
                                            }
                                        },
                                },
                                new StackLayout
                                {
                                    Orientation = StackOrientation.Horizontal,
                                    HorizontalOptions = LayoutOptions.FillAndExpand,
                                    Children =
                                        {
                                            new Label
                                            {
                                                Text = _leftLabel,
                                                HorizontalOptions = LayoutOptions.FillAndExpand,
                                                FontSize = 15
                                            },
                                            new Label
                                            {
                                                Text = _rightLabel,
                                                HorizontalOptions = LayoutOptions.End,
                                                FontSize = 15
                                            }
                                        },
                                    IsVisible = (_leftLabel != null || _rightLabel != null) ? true : false
                                }
                            }
                    });